File: Number of NGEs to the Right/Brute Force.java
import java.util.*;

public class Main {
    
    public static int countNGE(int[] arr, int idx) {
        
        int count = 0;
        
        for(int i=idx + 1;i<arr.length;i++) {
            if(arr[i] > arr[idx]) count++;
        }
        
        return count;
    }
    
    public static void main(String[] args) {
        Scanner scn = new Scanner(System.in);
        int n = scn.nextInt();
        int[] arr = new int[n];
        
        for(int i=0;i<n;i++) {
            arr[i] = scn.nextInt();
        }
        
        int Q = scn.nextInt();
        
        while(Q-- > 0) {
            int idx = scn.nextInt();
            int res = countNGE(arr,idx);
            System.out.println(res);
        }
        
        scn.close();
    }
}

File: Number of NGEs to the Right/Brute Force.cpp
#include <bits/stdc++.h>
using namespace std;

int countNGE(vector<int>& a, int index)
{
	int count = 0, N = a.size();
	for (int i = index + 1; i < N; i++)
		if (a[i] > a[index])
			count++;

	return count;
}

int main()
{
	int N;
	cin >> N;

    vector<int> arr(N);
	
	for(int i=0;i<N;i++) {
	    cin >> arr[i];
	}
	
	int Q;
	cin >> Q;
	
	while(Q--) {
	    int idx;
	    cin >> idx;
	    
	    int res = countNGE(arr,idx);
	    cout<< res << "\n";
	}
}
